pkg/operator/kube_cloud_config/controller.go

```diff
@@ -154,3 +151,12 @@ func asIsTransformer(input *corev1.ConfigMap, sourceKey string, _ *configv1.Infr
 
 	return output, nil
 }
+
+// cloudConfigTransformers returns all configured cloud transformers
+func cloudConfigTransformers() map[configv1.PlatformType]cloudConfigTransformer {
```

Comment:
make this a global var. No need to do this dynamically.

Reply:
Apparently, global maps in golang can't be initialized. I think it makes more sense and is cleaner. If global, then it will need to be initialized somewhere. Since it is used in a few places (and possibly more in the future), this will begin look uglier IMO. Let me know your thoughts.